app/assets/stylesheets/application.scss

```diff
@@ -47,10 +47,42 @@
 }
 
 .navbar-skinny {
-  min-height: 1.5em;
+  min-height: 2.5em;
 }
 
 .add-doc {
-  width: 3em;
+  width: 1.5em;
+  height: 1.5em;
+  padding-right: .5rem;
   display: inline-block;
 }
+
+.btn-secondary {
+  color: $white;
+}
+
+.view-btn {
+  margin-left: .75rem;
+}
+
+.share-btn {
+  margin-right: .75rem;
+}
+
+.cancel-btn {
+  margin-right: .75rem;
+}
+
+.single-view-btn {
+  margin-top: 1rem;
+  position: relative;
+}
+
+.form-control {
+  font-size: .75rem;
+}
+
+.form-share {
+  margin-right: 16rem;
+  margin-left: 16rem;
```

Comment:
I think this is how you're ending up with this:

<img width="392" alt="Screen Shot 2019-09-11 at 1 01 26 PM" src="https://user-images.githubusercontent.com/1615322/64660724-c6adb800-d495-11e9-923b-102f2a692159.png">

a better approach would be giving the button a parent with classes `d-flex justify-content-center`


for future reference, check http://howtocenterincss.com/ and https://css-tricks.com/snippets/css/a-guide-to-flexbox/